metabase/metabase
# feat(sdk): apply user interface color overrides to the sdk

Closes https://github.com/metabase/metabase/issues/42705

### Description

Applies the user interface colors in the enterprise appearance settings as the SDK's default value.

### How to verify

- Go to the `test-sdk-colors` branch in the demo app
- Go to `AppProvider.tsx`, and ensure that everything in `colors` are commented out
- Take note of the brand colors in the admin appearance settings
- The loading spinner should take the brand color from appearance settings.
- Try to override the `brand` color in the SDK, the sdk color should now take precedence.

### Caveat

- The filter should take the filter color not the brand color, but this is out-of-scope of this PR and @deniskaber is fixing this in his PR.

### Demo

<img src="https://github.com/metabase/metabase/assets/4714175/caa28494-5d9e-436e-9e5b-da7978d3af33" width="300">

<img src="https://github.com/metabase/metabase/assets/4714175/74c0803f-d03a-459b-81e2-9547339389e9" width="300">

<img src="https://github.com/metabase/metabase/assets/4714175/e202f0bb-4c73-437d-be6e-53d7464caed3" width="300">

### Checklist
